Library/MessageGenerator.swift

```diff
@@ -156,7 +156,7 @@ struct MessageGenerator {
     }
 
     private func shorterTomorrowLessThanAMinuteMessage(for hashValue: UInt32) -> Message {
-        let index = Int(hashValue % UInt32(self.shorterTomorrowLessThanAMinuteMessages.count))
+        let index = Int(hashValue % UInt32(self.shorterTomorrowLessThanAMinuteMessages.count - 1))
```

Comment:
why do all these indexes need to be adjusted?

Reply:
Was getting out-of-bounds crashes. 🤔

Reply:
On everything? Is this just since updating to the new JS lib?

Reply:
Not sure 🤔

Reply:
Anything you can do to test these and make sure it's still getting proper dates and not off-by-one errors?

Reply:
I'll gladly look into it when I'm back 👍